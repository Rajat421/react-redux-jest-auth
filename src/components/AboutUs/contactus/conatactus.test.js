/**
 * Created by Counter on 7/2/2017.
 */
import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import Contact from './contactus'

test('Contact component should render as expected',()=>{
    const component =shallow(<Contact/>);
    const tree =toJson(component);
    expect(tree).toMatchSnapshot()
});