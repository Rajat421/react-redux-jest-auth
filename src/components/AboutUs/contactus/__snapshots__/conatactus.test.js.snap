// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component should render as expected 1`] = `
<div>
  <section
    className="contactus section_padding"
    style={
      Object {
        "display": "table",
        "paddingTop": "47px",
      }
    }
  >
    <h3
      className="heading_h3"
    >
      Contact Us
    </h3>
    <p
      className="text_paragraphs"
    >
      Get in Touch!
      <br />
      We are here to answer any questions you may have about our experiences. Reach out to us and weâ€™ll respond as soon as we can.
      <br />
      We are available anytime for a Talk. So, Just stop by for a cup of coffee.
    </p>
    <div
      className="contactform"
    >
      <form
        action=""
        method=""
        name=""
      >
        <input
          name="name"
          placeholder="Enter Your Name"
          type="text"
        />
        <input
          name="useremail"
          placeholder="Enter Email Address"
          type="text"
        />
        <textarea
          placeholder="Enter Message"
        />
        <input
          type="submit"
          value="SEND"
        />
      </form>
    </div>
  </section>
</div>
`;
